src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,5 +1,63 @@
 package core.basesyntax;
 
 public class Plane {
+    private String planeName;
+    private String planeModel;
```

Comment:
```suggestion
    private String model;
```
`plane` word is redundant here. You already know that this is a property of Plane